TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmode/teleop/PositionTelemetry.java

```diff
@@ -0,0 +1,49 @@
+package org.firstinspires.ftc.teamcode.opmode.teleop;
+
+import com.acmerobotics.dashboard.FtcDashboard;
+import com.acmerobotics.dashboard.config.Config;
+import com.acmerobotics.dashboard.telemetry.MultipleTelemetry;
+import com.qualcomm.hardware.bosch.BNO055IMU;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+import org.firstinspires.ftc.teamcode.api.GyroTracker;
+import org.firstinspires.ftc.teamcode.api.Robot;
+import org.firstinspires.ftc.teamcode.api.config.Naming;
+import org.firstinspires.ftc.teamcode.api.hw.Gyroscope;
+
+@Config
+@TeleOp(name="Position Telemetry", group=Naming.OPMODE_GROUP_UTIL)
+public class PositionTelemetry extends LinearOpMode {
+    @Override
+    public void runOpMode() throws InterruptedException {
+        telemetry = new MultipleTelemetry(telemetry, FtcDashboard.getInstance().getTelemetry());
+        Robot.gyro1 = new Gyroscope(hardwareMap.get(BNO055IMU.class, "imu 1"), "gyro1");
```

Comment:
can we use the naming file for this?

Reply:
That is out of the scope of the PR, maybe do a different commit?